tests/test_project.py

```diff
@@ -2511,6 +2521,123 @@ def test_raise_exception_in_hook_cmd(self):
         assert "RuntimeError" in error_output
 
 
+class TestHooksLogOperationSetUp(TestHooksSetUp):
+    project_class = define_hooks_logging_project._HooksLogOperationsProject
+    entrypoint = dict(
+        path=os.path.realpath(
+            os.path.join(os.path.dirname(__file__), "define_hooks_logging_project.py")
+        )
+    )
+
+    ON_START_MSG = "Starting execution of operation '{}'."
+    SUCCESS_MSG = "Successfully finished execution of operation '{}'."
+    EXCEPTION_MSG = "Execution of operation '{}' failed with error"
```

Comment:
I know that the messages are not defined here, but commenting here for brevity. I would suggest to use a more consistent logging format to make it easier to parse those log files (programmatically). Something like:
```
Operation '{}' started.
Operation '{}' finished.
Operation '{}' failed with error '{}'.
```